ee/desktop/runtime/runner_linux.go

```diff
@@ -7,10 +7,6 @@ import (
 	"fmt"
 )
 
-func (r *DesktopUsersProcessesRunner) runConsoleUserDesktop() error {
+func (r *DesktopUsersProcessesRunner) runDesktopNative() error {
```

Comment:
I find `native` somewhat misleading -- I usually use it to refer to native code and/or tools. And launcher is basically all native. (There's no emulation or VM things). I think this is about invoking it as a user? `runDesktopAsUser`, or even the old name.

Reply:
I can see that ... maybe `runDesktopForOs` or `runDesktopForPlatform` ? I kinda wanted something call out that this func has a different definition for macos, windows, & linux, but I can't think of a good name so went back to runConsoleUserDesktop().

Reply:
I would't worry about that being clear in the naming.